### ISSUE TITLE
[Bug]: MO oom when run looply tpch 10g queries for about 1 hour

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#6527]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):b9c225f4f44dcd62d6f577394d705b6aed55188e
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

[1942275.993647] [3486277]  1028 3486277    22573      245   151552        0             0 mysql
[1942275.993650] oom-kill:constraint=CONSTRAINT_NONE,nodemask=(null),cpuset=/,mems_allowed=0-1,global_oom,task_memcg=/user.slice/user-1028.slice/session-414.scope,task=mo-service,pid=3466589,uid=1028
[1942275.993934] Out of memory: Killed process 3466589 (mo-service) total-vm:95320364kB, anon-rss:77996632kB, file-rss:88kB, shmem-rss:0kB, UID:1028 pgtables:170544kB oom_score_adj:0
[1942282.081041] oom_reaper: reaped process 3466589 (mo-service), now anon-rss:0kB, file-rss:0kB, shmem-rss:0kB
[sudong@mo-srv-129 matrixone]$ 

### Steps to Reproduce

```Markdown
as describe by issue title
```

**Comment by nnsgmsone** [comment]
fixed by #6535

**Comment by aressu1985** [comment]
fixed:c12eb45b1314e80f11fe8883055982c8ab0ca7fd
